notebooks/ExamplePlots.ipynb

```diff
@@ -11,7 +11,1066 @@
   },
```

Comment:
I must has accidentially committed changes to this notebook. That was not intentional. I've run it a couple of times to check that I did not mess up the matplotlib plotting but there should be no change in this PR. I'll have to google some git-fu to reverse it out again.